File: scripts/todoitem/put_todoitem.lua
timestamp_now = os.time(os.date("!*t"))
body = '{\"title\": \"test\", \"exp\":' .. timestamp_now + math.random(1, 10000) .. '}'

f = io.open(".token", "r")
token = f:read "*a"
f:close()

wrk.method = "PUT"
wrk.body = body
wrk.headers["Content-Type"] = "application/json"
wrk.headers["Authorization"] = "Bearer " .. token

File: scripts/todoitem/put_todoitem.js
import http from 'k6/http';

const url = '/todoitem';

export const options = {
    vus: 10,
    duration: '1s',
};

export default function () {
    const headers = {
        'Content-Type': 'application/json',
        'Authorization': 'Bearer '
    };
    const data = { title: 'test', exp: Math.floor(Date.now() / 1000) + Math.floor(Math.random() * 10000) };

    const res = http.put(url, JSON.stringify(data), { headers: headers });

    // console.log(JSON.parse(res.body).json.name);
}
